template.go

```diff
@@ -75,16 +87,15 @@ func (r *{{$typename}}) Scan(i interface{}) error {
 	case []byte:
 		return r.UnmarshalJSON(t)
```

Comment:
I'm not sure what database types, if any, gorp will pass as a `[]byte`. I think we should have both branches unless we never saw anything going through the `[]byte` case, in which case let's remove it.